### PR TITLE
Enhance screenshot handling

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
TODO 
- [x] enable it on drone 

- do not run all screenshot color/mode combination per each build
- instead run it once a day via drone (there is a cron command for it)
- have a summary then at https://www.kaminsky.me/nc-dev/android-screenshot-summary/2021-07-16/summary.html

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
example results you showed my looked nice @tobiasKaminsky 👍

**Comment by tobiasKaminsky** [comment]
Currently this might fail on screenshot blue-light job, but we can improve on this then there.
Same for all screenshot summary.

File: scripts/runAllScreenshotCombinations [hunk 1]
```diff
@@ -16,12 +16,12 @@ fi
 classMethod=$3
 
 resultCode=0
-while read line
+grep -v "#" scripts/screenshotCombinations | while read line
```
Comment (AlvaroBrey) [inline scripts/runAllScreenshotCombinations:19]: After seeing the `screenshotCombinations` file, doesn't this mean that only the `light blue` and `dark lightgreen` screenshot tests will be actually ran in drone?

Comment (tobiasKaminsky) [reply scripts/runAllScreenshotCombinations:19]: Right now yes. Sorry for not mentioning it. My idea was that we first get those two "green" and running reliable and then add the other colors.
To not forget what our "goal" is, I had the idea with commenting them.

**Review by AlvaroBrey — APPROVED** [review]

**Pull request merged**